utest.h

```diff
@@ -1202,7 +1255,21 @@ int utest_main(int argc, const char *const argv[]) {
 
     ns = utest_ns();
     errno = 0;
+#if defined(__cplusplus)
```

Comment:
One thing to note about this code - this will **break** if you have `UTEST_MAIN` in a C file, because this won't be defined.

Reply:
I don't think it will break anything, just not catch unexpected exceptions. This is unrelated to the new exception testing macros.